src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.hpp

```diff
@@ -109,6 +109,27 @@ class jit_exp_emitter : public jit_emitter {
     void register_table_entries() override;
 };
 
+class jit_floor_emitter : public jit_emitter {
+    public:
+        jit_floor_emitter(jit_generator* host, cpu_isa_t host_isa, const element::Type exec_prc = element::f32);
+        jit_floor_emitter(jit_generator* host, cpu_isa_t host_isa, const std::shared_ptr<ov::Node>& node);
+    
+        size_t get_inputs_num() const override;
+    size_t aux_vecs_count() const override;
+    size_t aux_fp_gprs_count() const override;  
+    
+    static std::set<std::vector<element::Type>> get_supported_precisions(
+        const std::shared_ptr<ov::Node>& node = nullptr);
+
+    private:
+        void emit_impl(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const override;
+    
+        template <cpu_isa_t isa>
+        void emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const;
+    
+        void register_table_entries() override;
+};
```

Comment:
Minor: please fix indents

```suggestion
class jit_floor_emitter : public jit_emitter {
public:
    jit_floor_emitter(jit_generator* host, cpu_isa_t host_isa, const element::Type exec_prc = element::f32);
    jit_floor_emitter(jit_generator* host, cpu_isa_t host_isa, const std::shared_ptr<ov::Node>& node);
    
    size_t get_inputs_num() const override;
    size_t aux_vecs_count() const override;
    size_t aux_fp_gprs_count() const override;  
    
    static std::set<std::vector<element::Type>> get_supported_precisions(
        const std::shared_ptr<ov::Node>& node = nullptr);

private:
    void emit_impl(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const override;
    
    template <cpu_isa_t isa>
    void emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const;
    
    void register_table_entries() override;
};
```